packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [3.1.2](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.1...flutter-3.1.2) (2022-06-06)
+
+### Bug Fixes
+
+* unhandled exception in `ParseRelation`, type `ParseObject` is not a subtype of type ([#696](https://github.com/parse-community/Parse-SDK-Flutter/issues/696))
+* error in progress callback ([#679](https://github.com/parse-community/Parse-SDK-Flutter/issues/679))
+* incorrect return type when calling `first()` ([#661](https://github.com/parse-community/Parse-SDK-Flutter/issues/661))
+* error in `ParseLiveListWidget` when enabling `lazyloading` ([#653](https://github.com/parse-community/Parse-SDK-Flutter/issues/653))
+* fixes an error that occurs when the logout method is called and the user does not have an access token, for example, after creating an account that must be verified
```

Comment:
issue reference missing

Reply:
which issue? if it is the latter, there is no open issue.
I commented on this sometime before.
I found and fixed this issue before the package was transferred to Parse. It wasn't standard to have to open an issue for every fix in the code.

Reply:
Could you link the PR if there is no issue?

Reply:
Why is this resolved?

Reply:
@RodrigoSMarques Could you just add a reference here, so we can close this point?